integration/integration_test__ena.py

```diff
@@ -0,0 +1,31 @@
+import os
+import time
+
+from api.ingest import IngestAPI
+from ena.ena_api import EnaApi
+from unittest import TestCase
+
+from ena.util import write_xml
+
+ENA_API_DEV = 'https://wwwdev.ebi.ac.uk/ena/submit/drop-box/submit/'
+INGEST_ARCHIVER_API_STAGING = "https://archiver.ingest.staging.archive.data.humancellatlas.org"
+INGEST_API = 'https://api.ingest.staging.archive.data.humancellatlas.org/'
+
+
+class SampleRunTest(TestCase):
```

Comment:
This test has not got any assertions and I am not sure its name is correct.

Reply:
So this integration test is WIP, but I don't wanna remove the test files, is it okay if we push these files? I've added some notes and TODO comments for now.

Reply:
I could just remove them if you prefer that and just put them in a branch.

Reply:
i deleted the integration directory and stored them in this branch task/submit-10x-fastq-files-integration-tests for reference